src/services/errors/http-error-handler.ts

```diff
@@ -27,7 +29,8 @@ export class HttpErrorHandler {
   }
 
   handle(err: AxiosError | Error) {
-    const error = this.mapError(err);
-    throw new HttpException(error, error.code);
+    const errPayload = this.mapError(err);
+    this.logger.error(errPayload);
+    throw new HttpException(errPayload, errPayload.code);
```

Comment:
Question: where does this `errPayload.code` come from? In case of the chains endpoint, from the config-service?

Maybe it would make sense to have a centralized error registry where you would create your own errors with distinct CGW-only error codes, and send that to the client.

Reply:
At this point the error could come from any http service we're calling (both Config and Transaction services). I think having a centralised registry it's a good idea, probably this will be done in a further development step, we're keeping it simple for now but we'll do it. In fact this explicit CGW-only error codes is something you already have in place in the old CGW implementation, right?

Reply:
AFAIK, we only received 42 and 1337 error codes in the old CGW.

Reply:
The only use case I know so far is when we need to check if a safe exists or just failed to load.

But obviously, if we had a registry of backend error codes with explanations, we could potentially handle errors better on the frontend end. Or even display them to the user directly.

Reply:
This is something that we can (and should) review alongside the clients' teams. 

If the clients' use the current error codes then we will match them on an initial phase so that there are no surprises on your end. Either way as you mentioned, we should align this either for the initial release or one that comes after that 👍